python_modules/dagster-airflow/dagster_airflow_tests/test_run_airflow_dag.py

```diff
@@ -59,6 +59,14 @@ def test_unit_run_airflow_dag_steps(airflow_test, dags_path):
         assert 'EXECUTION_PLAN_STEP_SUCCESS' in str(res)
 
         for step_output in step.step_outputs:
-            assert 'for output {output_name}'.format(output_name=step_output.name) in str(res)
+
```

Comment:
@mgasner not sure what is going on here

Reply:
we get the raw logs back from the docker container execution, this asserts that the logs tell us each output was actually returned